.eslintrc

```diff
@@ -1,9 +1,13 @@
+plugins:
+  - jsx-a11y
 extends:
   - standard
   - standard-react
+  - plugin:jsx-a11y/recommended
 parser: babel-eslint
 rules:
   object-curly-spacing: [ error, always ]
   react/prop-types: off
+  jsx-a11y/rule-name: 2
```

Comment:
I think this is incorrect.
`rule-name` is meant to be a placeholder not real name of a rule.

Since you extend a base configuration `jsx-a11ly/recommended`, you don't need to specify rules.